### DISCUSSION TITLE
How can I see end of life or support of version?

### DISCUSSION THREAD

**Discussion by shahankitb-997** [katspaugh/wavesurfer.js#3717] [Q&A] [answered]
For example : https://endoflife.date/nodejs

I am looking for https://www.npmjs.com/package/wavesurfer.js/v/7.7.13 or any of the version end of life or support of it.

So on basis of that we can decide, we can go with that particular version or not.

I am not able to find it.

**Comment by katspaugh** [comment, marked as answer]
Only the latest major version is supported atm. It's not an official support, only informal Q&A in this forum.
